src/styles/main.scss

```diff
@@ -0,0 +1,60 @@
+@import '../utils/variables';
+
+* {
+  margin: 0;
+}
+
+body {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+  max-width: ($col-count * $block) + $padding-calc;
+
+  &__day {
+    box-sizing: border-box;
+    height: $block;
+    width: $block;
+
+    border: 1px solid #000;
+    background-color: #eee;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition: 500ms;
+
+    @for $num-of-day from 1 through 31 {
+      &--#{$num-of-day}::before {
+        content: '#{$num-of-day}';
+        color: #000;
+        font-family: Arial, sans-serif;
+        font-size: 30px;
```

Comment:
Consider moving common styles to a general class instead of generating them for each day in the cycle